README.md

```diff
@@ -24,14 +16,44 @@ Its meetings and membership are largely focused on the direction, design, and wo
 
 The following subprojects are owned by the Working Group:
 
-{{
-
-* template-project
-  * Description: Brief description of project. Remove this item and add new projects using this format.
+* Organization Infrastructure
+  * Description: Repositories to manage the working group itself - these are not products for users
+  * Repositories:
+    * https://github.com/RobotWebTools/community
+    * https://github.com/RobotWebTools/robotwebtools.github.io
+    * https://github.com/RobotWebTools/starter-template
+    * https://github.com/RobotWebTools/rwt_dockers
+    * https://github.com/RobotWebTools/robot_web_tools
+* `roslibjs`
+  * Description: ROS 1 Javascript language client - for use in the browser
+  * Repositories:
+    * https://github.com/RobotWebTools/roslibjs
+    * https://github.com/RobotWebTools/roslibjs_integration_tests
```

Comment:
This was last updated 2014 - it seems dead. Perhaps we should remove it

Reply:
Yes. roslibjs_integration_tests can be removed.